### PR TITLE
fix README.md

### PR DESCRIPTION
**Pull request by Yiabiten**
remove duplicate 'be'

### CODE REVIEW

**Comment by kuba** [comment]
Welcome to the project, thanks a lot for catching this! :)

File: README.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ sudo ./bootstrap.sh
 
 ## Examples
 
-Have a look into `./examples/`.
\ No newline at end of file
+Have a look into `./examples/`.
```
Comment (kuba) [inline README.md:64]: what's up with this line?

Comment (kuba) [inline README.md:64]: ah, it's the newline, which was missing in `master`

**Comment by Yiabiten** [comment]
You're welcome :)

**Pull request merged**